### PR TITLE
Genetics bundle

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

Farsight: lets you see further
Fast twitch: lets you move slightly faster
Signal Jammer: lets you be invisible to the AI

genetics kinda meh ever since a ton of good powers got removed. give them some more to make up for it and maybe clockcult and malf will bother targeting them again

# Spriting

# Wiki Documentation

Farsight: lets you see further
Fast twitch: lets you move slightly faster
Signal Jammer: lets you be invisible to the AI
# Changelog

:cl:  
rscadd: New genetics powers
rscadd: Farsight: lets you see further
rscadd: Fast twitch: lets you move slightly faster
rscadd: Signal Jammer: lets you be invisible to the AI
/:cl:

### CODE REVIEW

**Comment by Annoying-plant** [comment]
would be nice to see one that provides a slow passive heal although I reckon it would need to cost a lot of stability something you would be willing to do?

**Comment by Annoying-plant** [comment]
but i really like the idea of more genetics its kind of only space adapt people want now although bone density and and the other bone one are really good hopefully ill play more gene with we get more genetics some more cool stuff

**Comment by Simplehorror** [comment]
> 

yeah i will add the regen

File: code/modules/mob/living/carbon/examine.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@
 		else if(InCritical())
 			. += "[t_His] breathing is shallow and labored."
 
-		if(digitalcamo)
+		if(digitalcamo && mind?.has_antag_datum(/datum/antagonist/changeling))
```
Comment (tattax) [inline code/modules/mob/living/carbon/examine.dm]:
```suggestion
		if(digitalcamo && mind?.has_antag_datum(/datum/antagonist/changeling) || mind?.has_antag_datum(/datum/antagonist/bloodsucker))
```

**Pull request closed without merging**